h2o-core/src/main/java/water/rapids/FeatureImportance4BBM.java

```diff
@@ -0,0 +1,129 @@
+package water.rapids;
+
+import hex.AUC2;
+import hex.Model;
+//import hex.VarImp;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.util.*;
+
+import java.util.*;
+
+
+public class FeatureImportance4BBM /* extends VarImp */ {
+    
+    private Model _model;
+    public Frame _ouput;
+    private Frame _train_fr; // original dataset used to train model
+    private Frame _validate_dataset; // original dataset used to train model
+    private Frame _predcited_frame;
+
+    private boolean _data_in_holdOut_form; // do I have the data split in train and validation, if not spliting will happen
+
+    private String _responce_col;
+
+    private AUC2 og_auc;
+
+    private double original_mse;
+    private double sq_err;
+            
+    public FeatureImportance4BBM(Model model) {_model = model;}
+
+    public FeatureImportance4BBM(Model model, Frame data_set, Frame prediction_frame) {
+        _model = model;
+        _train_fr = data_set;
+        _data_in_holdOut_form = false;
+        _validate_dataset = null;
```

Comment:
If you want to use the validation dataset, you also should think about cross-validation datasets... Maybe for the beginning, only training dataset is enough.